source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,75 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.findOne()`` method. 
+
+..
+   method
+
+The ``findOne()`` method takes a query document that specifies the
+subset of your collection that you would like to retrieve. If you
+pass the ``findOne()`` method an empty document, MongoDB will retrieve your
+entire collection. For more information on documents in the MongoDB Go driver,
+see our :doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``findOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
+
+..
+   options
+
+You can specify additional options for your ``findOne()`` method call by passing
+the ``findOne()`` method a ``*FindOneOptions`` value. For example, your
```

Comment:
Thank you for the note. I agree it makes sense to lead off with an introduction to the use case of the feature rather than the implementation details of the feature. Leading with implementation details is a pattern I have struggled with, and I will try to correct it. Apologies. I have reordered the information in the paragraph such that it leads with the use case.

I have rephrased the sentence to avoid saying `FindOne()` twice.

Fix: https://github.com/mongodb/docs-golang/pull/2/commits/dc0fe9b085af68be2d0e55c7c5edaf762c099486